docs/Containerfile.5.md

```
@@ -0,0 +1,511 @@
+% "CONTAINERFILE" "5" "Aug 2021" "" "Container User Manuals"
+
+# NAME
+
+Containerfile(Dockerfile) - automate the steps of creating a container image
+
+# INTRODUCTION
+
+The **Containerfile** is a configuration file that automates the steps of creating a container image. It is similar to a Makefile. Container engines (Podman, Buildah, Docker) read instructions from the **Containerfile** to automate the steps otherwise performed manually to create an image. To build an image, create a file called **Containerfile**.
+
+The **Containerfile** describes the steps taken to assemble the image. When the
+**Containerfile** has been created, call the `buildah bud`, `podman build`, `docker build` command,
+using the path of context directory that contains **Containerfile** as the argument. Podman and Buildah default to **Containerfile** and will fall back to **Dockerfile**. Docker only will search for **Dockerfile** in the context directory.
+
+
+**Dockerfile** is an alternate name for the same object.  **Containerfile** and **Dockerfile** support the same syntax.
+
+# SYNOPSIS
+
+INSTRUCTION arguments
+
+For example:
+
+  FROM image
+
+# DESCRIPTION
+
+A Containerfile is a file that automates the steps of creating a container image.
+A Containerfile is similar to a Makefile.
+
+# USAGE
+
+  ```
+  buildah bud .
+  podman build .
+  ```
+
+  -- Runs the steps and commits them, building a final image.
+  The path to the source repository defines where to find the context of the
+  build.
+
+  ```
+  buildah bud -t repository/tag .
+  podman build -t repository/tag .
+  ```
+
+  -- specifies a repository and tag at which to save the new image if the build
+  succeeds. The container engine runs the steps one-by-one, committing the result
+  to a new image if necessary, before finally outputting the ID of the new
+  image.
+
+  Container engines re-use intermediate images whenever possible. This significantly
+  accelerates the *build* process.
+
+# FORMAT
+
+  `FROM image`
+
+  `FROM image:tag`
+
+  `FROM image@digest`
+
+  -- The **FROM** instruction sets the base image for subsequent instructions. A
+  valid Containerfile must have **FROM** as its first instruction. The image can be any
```

```suggestion
  valid Containerfile must have **FROM** as its first instruction. The image can be any
```
An `ARG` can be the first line too...

---

Are you going to adjust this?